### ISSUE TITLE
[Adaptive Cards-InputsWithValidation.json]: Screen reader is not announcing the "Current Date" information on while navigating to the current today date picker control on the 'InputsWithValidation.json' card.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9226]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Screen Reader: Narrator/NVDA

**Repro steps:**
1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the Adaptive Cards Visualizer application.
3. Navigate to the 'InputsWithValidation.json' control present on left navigation using tab key and activate it using enter key.
4. 'InputsWithValidation.json' adaptive card will appear on right sight of the pane.
5. Navigate to the 'Due Date' combo box field using tab key and activate using enter key.
6. Now navigate to the current today date using arrow keys.
7. Verify that, screen reader is announcing the "Current Date" information on while navigating to the current date picker control on the 'InputsWithValidation.json' card or not.

**Actual Result:**
Screen reader is not announcing the "Current Date" information on while navigating to the current today date picker control on the 'InputsWithValidation.json' card.

**Note:**
Same issue replicates for the 'Enter the due date' calendar control present on the 'ActivityUpdate.Json' card.

**Observation:**
Same issue replicates with NVDA.

**Expected Result:**
Screen reader should properly announce the "Current Date" information on while navigating to the current today date picker control on the 'InputsWithValidation.json' card.

**User Impact:**
Visually impaired users who rely on screen readers will be impacted if the "Current Date" is not announced when navigating to the current today date picker control. Without this information, it may leading to confusion or incorrect selections. This affects their ability to interact with the form independently and efficiently. It also reduces overall accessibility and may result in non-compliance with accessibility standards.

**MAS Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/1f01c2d9-a5d3-40d6-b6bf-83b21dbd1234

### Card JSON

```JSON
N/A
```
